test/controllers/static_pages_controller_test.rb

```diff
@@ -0,0 +1,37 @@
+require "test_helper"
+
+class StaticPagesControllerTest < ActionDispatch::IntegrationTest
+  
+  def setup
+    @base_title = "Ruby on Rails Tutorial Sample App"
+  end
+  
+  test "should get home" do
```

Comment:
At DocFox, we have a specific way of naming controller tests.  They are of the format:

`ACTION #method descriptive test name`

So this one, please change to something like:

`test "GET #home renders the home page"`